resources/monitoring/charts/grafana/values.yaml

```diff
@@ -130,13 +147,15 @@ podPortName: grafana
 ## ref: http://kubernetes.io/docs/user-guide/services/
 ##
 service:
+  enabled: true
   type: ClusterIP
   port: 80
-  targetPort: 3000
+  targetPort:
+    3000
     # targetPort: 4181 To be used with a proxy extraContainer
   annotations: {}
   labels: {}
-  portName: http-service
+  portName: service
```

Comment:
The renaming was done to make it work with the newer version of Istio in [this PR](https://github.com/kyma-project/kyma/pull/6639/files#diff-464842021c732f81d6594a12d0ebf9f0b68874822c8db4c88ad8e3e4042947a7R120). Please make sure this won't break it again. I think @Tomasz-Smelcerz-SAP can verify which one is correct.